crates/ludtwig/src/check/rules/twig_logic_same_as.rs

```diff
@@ -0,0 +1,76 @@
+use ludtwig_parser::syntax::typed::{AstNode, TwigBinaryExpression};
+use ludtwig_parser::syntax::untyped::{SyntaxKind, SyntaxNode};
+
+use crate::check::rule::{CheckResult, Rule, RuleExt, RuleRunContext, Severity};
+
+pub struct RuleTwigLogicSameAs;
+
+impl Rule for RuleTwigLogicSameAs {
+    fn name(&self) -> &'static str {
+        "twig-logic-same-as"
+    }
+
+    fn check_node(&self, node: SyntaxNode, _ctx: &RuleRunContext) -> Option<Vec<CheckResult>> {
+        let binary = TwigBinaryExpression::cast(node.clone())?;
+        let binary_expr_op = binary.operator()?;
+
+        if binary_expr_op.kind() == SyntaxKind::TK_TRIPLE_EQUAL
+            || binary_expr_op.kind() == SyntaxKind::TK_DOUBLE_EQUAL
```

Comment:
Actually double `==` is valid twig, see https://twig.symfony.com/doc/3.x/templates.html#comparisons

Only `===` is not supported by official twig and twig provides the `same as` syntax to allow for stricter equal checks.

I would prefer to not suggest replacing `==` with `same as`, at least that wasn't the initial intention (using `===` should be a hard error while using `==` is maybe not strict enough / a suggestion to make it stricter). Or at least that should be in a different rule to support ignoring it